styles/mixins/patterns/_molecules.scss

```diff
@@ -59,3 +59,33 @@ $-space: var.$layout-space;
     @include layout.vertical-rhythm($-space * 0.5);
   }
 }
+
+// A responsive sized and positioned container for modal content
+@mixin modal {
+  // The modal container is positioned horizontally- and vertically-centered
+  @include layout.fixed-centered;
+
+  // For narrower viewports/default:
+  // - positioning: centered vertically and horizontally in the viewport
+  // - sizing: fills up most of the available horizontal space
+  width: 90vw;
+  max-width: 90vw;
+  max-height: 90vh;
+
+  // For wider viewports:
+  // - positioning: keep horizontal centering but bring top nearer to top
+  //     of viewport.
```

Comment:
I noted this in another comment, but I think we might want to make the vertical centering behavior dependent on the height rather than the width of the viewport.

Reply:
It's good feedback. I'm working on splitting the media-query-based differentiation out into two now:

* Screens that are _wide_ enough: set a reasonable min- and max-width so that the modal isn't too dinky but also doesn't stretch way out
* Screens that are _tall_ enough: position the modal nearer to the top of the viewport

e.g. Narrow, but tall screen:

![image](https://user-images.githubusercontent.com/439947/120808673-4a83bf80-c517-11eb-90e9-171b6a03aeda.png)